include/llbuild/BuildSystem/BuildValue.h

```diff
@@ -52,15 +52,21 @@ class BuildValue {
 
     /// A value for a produced output whose command failed.
     FailedInput,
+      
+    /// A value for a produced output whose command was cancelled.
+    CancelledInput,
```

Comment:
I think this commit should technically bump `BuildSystemImpl::internalSchemaVersion`, since this effectively will change the binary encoding of values in existing databases. This will force a rebuild for anyone who runs with a database shared before and after this build (something we don't, unfortunately, have any test coverage of).